SP-Framework/spfx-toc/toc.yml

```diff
@@ -1,279 +1,276 @@
-- name: SharePoint reference
-  href: ~/overview/sharepoint.md
+- name: sp-application-base
+  uid: sp-application-base
   items:
```

Comment:
FYI toc.yml is a machine-generated file.  I will need to update the [api-documenter](https://github.com/Microsoft/web-build-tools/tree/master/libraries/api-documenter) tool to reflect your change here.